webapp/src/app.tsx

```diff
@@ -1533,9 +1548,15 @@ ${compileService && compileService.githubCorePackage && compileService.gittag ?
 
     startTutorial(tutorialId: string, tutorialTitle?: string) {
         pxt.tickEvent("tutorial.start");
-        core.showLoading(lf("starting tutorial..."));
-        this.startTutorialAsync(tutorialId, tutorialTitle)
-            .then(() => Promise.delay(500));
+        // Check for Internet access
+        if (!pxt.BrowserUtils.isOnline()) {
```

Comment:
use Cloud.isOnline

Reply:
Cloud.isOnline maintains a variable remembering if the cloud is online. going to use cloud.isNavigatorOnline for this one, as there's a possibility that the cloud could be online but the user just went offline.